Site/ASAB Maestro/Descriptors/elasticsearch.yaml

```diff
@@ -38,3 +38,56 @@ descriptor:
     xpack.security.transport.ssl.certificate_authorities: certs/ca.pem
     xpack.security.transport.ssl.certificate: certs/certificate.pem
     xpack.security.transport.ssl.key: certs/key.pem
+
+
+nginx:
+  https:
```

Comment:
@byewokko I did some remix of the Kibana endpoint and previous elastic endpoint... I'll be happy if you have a look. There might be some lines that are maybe not needed

Reply:
looks good overall, just see my comment on the cache key